src/camera/Camera.ts

```diff
@@ -149,9 +181,9 @@ abstract class Camera {
       return;
     }
 
-    const viewportSize = flicking.viewport.size;
+    const viewport = flicking.viewport;
     const cameraPosition = this._position;
-    const cameraSize = flicking.horizontal ? viewportSize.width : viewportSize.height;
+    const cameraSize = flicking.horizontal ? viewport.width : viewport.height;
```

Comment:
The logic to find the size of the viewport is often seen. How about making it a function?
`flicking.horizontal ? viewport.width : viewport.height;`

Reply:
That's right, I'm considering adding it as a `camera.size` maybe